app/config/config_test.yml

```diff
@@ -37,7 +37,7 @@ monolog:
 
 liip_functional_test:
     cache_sqlite_db: true
-    query_count.max_query_count: 50
+    #query_count.max_query_count: 50
```

Comment:
Lets just toss this - we're not hitting it now - and its about as bad as it can get for query count.  I say just take out this whole section.